docs/model/elasticsearch.md

```diff
@@ -41,13 +41,18 @@ Following product attributes are exported into Elasticsearch (i.e. the search or
 * in_stock (true/false value whether the product is in stock)
 * parameters (pairs of parameter IDs and parameter value IDs)
 * ordering_priority (priority number)
-* calculated_selling_denied (true/false value whether the product is already sold out)
+* calculated_selling_denied (calculated true/false value whether the product is already sold out)
+* selling_denied (true/false value whether the product can be sold)
+* availability (translation of products availability)
```

Comment:
we use singular in other bullets. `translation of product availability` will be better